tests/test_SR.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import print_function
+import unittest
+import mock
+import SR
+import xs_errors
+
+
+class TestSR(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def create_SR(self, cmd, dconf):
+        srcmd = mock.Mock()
+        srcmd.dconf = dconf
+        srcmd.params = {'command': cmd}
+        return SR.SR(srcmd, "some SR UUID")
+
+    def test_checkroot_no_device(self):
```

Comment:
Do these tests need to be asserting anything after the calls to check they functioned correctly?